fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -257,7 +260,17 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management)
         }
     }
 
+    private fun openAMO() {
+        (activity as HomeActivity).openToBrowserAndLoad(
+            searchTermOrURL = AMO_HOMEPAGE_FOR_ANDROID,
+            newTab = true,
+            from = BrowserDirection.FromGlobal,
```

Comment:
Is there a better `from` value? There is no direction for the manager view for the moment.

Reply:
I think global should fine and good enough for this use case :)